packages/@monocdk-experiment/assert/clone.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# clones @aws-cdk/assert into here
+scriptdir=$(cd $(dirname $0) && pwd)
+cd $scriptdir
+set -euo pipefail
+src="../../@aws-cdk/assert"
+rsync -av $src/lib/ lib/
+rsync -av $src/test/ test/
+
+files="README.md LICENSE NOTICE .npmignore jest.ts"
+
+for file in ${files}; do
+  cp $src/$file .
+done
+
+npx rewrite-imports ./lib/*.ts
```

Comment:
This could be one invocation:

```
rewrite-imports {lib,test}/*.ts jest.ts
```

I'm a little afraid that if we don't have project references on this file the rewriter might crawl into other packages if they have been symlinked-to-source... does that happen?

Reply:
I does, but I updated the rewriter to ignore anything that is not listed in the file list.